### PR TITLE
Bryanna/toast component

### PR DESCRIPTION
**Pull request by bry-gavino**
Made a Toast component with 2 different alert styles and 1 error style. _(see screenshots below)_

## Screenshots

Screen Shot 2020-12-02 at 6 50 40 PM
The comments for 'acceptedAlert', 'verifyEmailAlert', and 'userExistsError' are pretty self-explanatory. Underneath //general , this is where you can customize the Toast component's message and choose what style to use. 

**verticalPos?** and **horizontalPos?** are props you can pass in to place the Toast in different places of the screen.

The other props with ?s are passed in when you want to customize the Toast.

Customized Toast:
Screen Shot 2020-12-02 at 6 53 17 PM

The different Figma Toast designs:
Screen Shot 2020-12-02 at 6 53 34 PM
Screen Shot 2020-12-02 at 6 53 46 PM
Screen Shot 2020-12-02 at 6 54 01 PM

CC: @kwfk

### CODE REVIEW

**Comment by claalmve** [comment]
This PR is brilliant! Apologies it took so long to approve, but just made some minor type changes and code abbreviation. I played around with the different states and really liked how easy it was to use from both the user's and dev's standpoints. Great work!

**Review by claalmve — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -48,7 +48,8 @@
     "next-auth": "^3.1.0",
     "react": "16.13.1",
     "react-dom": "16.13.1",
-    "react-map-gl": "^5.2.9"
+    "react-map-gl": "^5.2.9",
+    "react-toastify": "^6.1.0"
```
Comment (claalmve) [inline package.json]: Is this package still needed?

**Review by claalmve — COMMENTED** [review]
My only comment was whether or not we still needed the `react-toastify` package, as it seems like the code is fully functional without it.

Otherwise, once/if that's addressed, the PR should be good to go to merge in!

**Comment by kwfk** [comment]
ah i had some changes that i'll push very soon

**Review by kwfk — CHANGES_REQUESTED** [review]
there are some changes i made before i will push soon

**Comment by kwfk** [comment]
image

**Review by kwfk — APPROVED** [review]
pushed changes

**Pull request merged**